test/libsolidity/InlineAssembly.cpp

```diff
@@ -137,22 +138,22 @@ BOOST_AUTO_TEST_CASE(smoke_test)
 
 BOOST_AUTO_TEST_CASE(simple_instructions)
 {
-	BOOST_CHECK(successParse("{ dup1 dup1 mul dup1 sub }"));
+	BOOST_CHECK(successParse("{ dup1 dup1 mul dup1 sub pop }"));
 }
 
 BOOST_AUTO_TEST_CASE(suicide_selfdestruct)
 {
-	BOOST_CHECK(successParse("{ suicide selfdestruct }"));
+	BOOST_CHECK(successParse("{ 0x01 suicide 0x02 selfdestruct }"));
```

Comment:
I wonder if we'd need to split it into `successParser()` and `successParseAndAnalyze()`?